### DISCUSSION TITLE
Under MAC arm64 --cxxopt=-mmacosx-version-min=14.2 gcc-11 error

### DISCUSSION THREAD

**Discussion by the-code-samurai-97** [bazelbuild/bazel#20698] [Q&A]
bazel - 7.0.0
compiler - gcc-11
when i build
error: `gcc-11: error: unrecognized command-line option '-mmacos-version-min=14.2'; did you mean '-mmacosx-version-min='?`

where should i change this ?

**Comment by keith** [comment]
The error is from gcc showing that you spelled the command line option wrong, you're missing the `x` in `macosx`

**Reply by sluongng** [reply]
I think it's caused by https://cs.opensource.google/bazel/bazel/+/master:tools/cpp/unix_cc_toolchain_config.bzl;l=1338;drc=044a14cca2747aeff258fc71eaeb153c08cb34d5

**Reply by keith** [reply]
Ah yes, sorry I assumed that was passed manually because of the title of this. We should fix this, I bet clang respects both still

**Reply by the-code-samurai-97** [reply]
Thanks! @keith and @sluongng , is there a beginner document for this toolchain, i am a beginner here

**Reply by keith** [reply]
@the-code-samurai-97 can you file an issue and fill out the template? now that I've thought about it more I think a repro case would be helpful

**Reply by keith** [reply]
here's a change to that flag https://github.com/bazelbuild/bazel/pull/20798 but i don't know how you got into this state

**Comment by iancha1992** [comment]
> bazel - 7.0.0 compiler - gcc-11 when i build error: `gcc-11: error: unrecognized command-line option '-mmacos-version-min=14.2'; did you mean '-mmacosx-version-min='?`
> 
> where should i change this ?

A fix for this issue has been included in [Bazel 7.1.0 RC1](https://releases.bazel.build/7.1.0/rc1/index.html). Please test out the release candidate and report any issues as soon as possible. Thanks!
